NoSuchAlgorithmException: Algorithm HmacPBESHA256 not available with 7.3.2 streams-demo

**Description**
On `7.3.2-post` branch, the `streams-demo` container is failing with a Schema Registry REST client issue:

```
Exception in thread "main" java.lang.RuntimeException: Error initializing the ssl context for RestService
	at io.confluent.kafka.schemaregistry.client.security.SslFactory.<init>(SslFactory.java:72)
	at io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient.<init>(CachedSchemaRegistryClient.java:184)
	at io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient.<init>(CachedSchemaRegistryClient.java:135)
	at io.confluent.kafka.serializers.AbstractKafkaSchemaSerDe.configureClientProperties(AbstractKafkaSchemaSerDe.java:82)
	at io.confluent.kafka.serializers.AbstractKafkaAvroSerializer.configure(AbstractKafkaAvroSerializer.java:53)
	at io.confluent.kafka.serializers.KafkaAvroSerializer.configure(KafkaAvroSerializer.java:50)
	at io.confluent.kafka.streams.serdes.avro.SpecificAvroSerializer.configure(SpecificAvroSerializer.java:58)
	at io.confluent.kafka.streams.serdes.avro.SpecificAvroSerde.configure(SpecificAvroSerde.java:107)
	at io.confluent.demos.common.wiki.WikipediaActivityMonitor.main(WikipediaActivityMonitor.java:139)
Caused by: java.io.IOException: Integrity check failed: java.security.NoSuchAlgorithmException: Algorithm HmacPBESHA256 not available
	at java.base/sun.security.pkcs12.PKCS12KeyStore.engineLoad(PKCS12KeyStore.java:2167)
	at java.base/sun.security.util.KeyStoreDelegator.engineLoad(KeyStoreDelegator.java:222)
	at java.base/java.security.KeyStore.load(KeyStore.java:1479)
	at io.confluent.kafka.schemaregistry.client.security.SslFactory$SecurityStore.load(SslFactory.java:163)
	at io.confluent.kafka.schemaregistry.client.security.SslFactory$SecurityStore.access$000(SslFactory.java:145)
	at io.confluent.kafka.schemaregistry.client.security.SslFactory.createSslContext(SslFactory.java:107)
	at io.confluent.kafka.schemaregistry.client.security.SslFactory.<init>(SslFactory.java:70)
	... 8 more
Caused by: java.security.NoSuchAlgorithmException: Algorithm HmacPBESHA256 not available
	at java.base/javax.crypto.Mac.getInstance(Mac.java:191)
	at java.base/sun.security.pkcs12.PKCS12KeyStore.engineLoad(PKCS12KeyStore.java:2145)
	... 14 more
```

This is likely caused by this JDK bug: https://bugs.openjdk.org/browse/JDK-8267701 and slight JDK 11 version variations between the `streams-demo` image build and the image that is building the trust stores:

```
docker run -it --rm --entrypoint /usr/bin/java cnfldemos/cp-demo-kstreams:0.0.10 -version
openjdk version "11.0.11" 2021-04-20 LTS
OpenJDK Runtime Environment Zulu11.48+21-CA (build 11.0.11+9-LTS)
OpenJDK 64-Bit Server VM Zulu11.48+21-CA (build 11.0.11+9-LTS, mixed mode)

docker run -it --rm --entrypoint /usr/bin/java confluentinc/cp-server:7.3.2 -version
openjdk version "11.0.18" 2023-01-17 LTS
OpenJDK Runtime Environment Zulu11.62+17-CA (build 11.0.18+10-LTS)
OpenJDK 64-Bit Server VM Zulu11.62+17-CA (build 11.0.18+10-LTS, mixed mode)
```

Fix is to re-build and push the `cp-demo-kstreams` image with a newer JDK version to match.

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1005,7 +1005,7 @@ services:
 
 
   streams-demo:
-    image: cnfldemos/cp-demo-kstreams:0.0.10
+    image: cnfldemos/cp-demo-kstreams:0.0.11
     restart: always
     depends_on:
       schemaregistry:

--- a/kstreams-app/build.gradle
+++ b/kstreams-app/build.gradle
@@ -14,7 +14,7 @@ plugins {
 }
 
 group = 'io.confluent'
-version = '0.0.10'
+version = '0.0.11'
 sourceCompatibility = "1.8"
 targetCompatibility = "1.8"
 
@@ -22,7 +22,7 @@ def props = new Properties()
 file("../env_files/config.env").withInputStream { props.load(it) }
 
 ext {
-  cpVersion = props.getProperty("CONFLUENT") ?: '6.2.0'
+  cpVersion = props.getProperty("CONFLUENT") ?: '7.3.2'
 }
 
 repositories {

